src/screenComponents/music.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MUSIC_H
+#define MUSIC_H
+
+#include "stringImproved.h"
+#include "threatLevelEstimate.h"
+
+class Music
+{
+private:
+    P<ThreatLevelEstimate> threat_estimate;
+    string music_file;
+    bool music_enabled;
+    bool threat_set_enabled;
+public:
+    Music(bool enabled);
+
```

Comment:
Adding a destructor `~Music()` which stops the music ensures that the music stops playing as soon as this object is destroyed. Needing less "manual" calling of stop to stop the music when screens are exited.